c2corg_api/views/route.py

```diff
@@ -52,3 +51,13 @@ def collection_post(self):
                           validators=[validate_id, validate_route_update])
     def put(self):
         return self._put(Route, schema_route)
+
+
+# FIXME: handle /routes/history/{id} ?
+@resource(path='/history/route/{id}', cors_policy=cors_policy)
```

Comment:
Don't forget to take the culture into account!
We only need the versions for a given culture.

Wouldn't it be more relevant to use ``/routes/history/{id}/{lang}`` instead? "history" is an action about the document type "routes", I'm not sure it should be the first element of the route => first comes the resources name, then the action and finally the parameters. Or the document reference (doc type > id > culture) and finally the action.